cpan/perlfaq/lib/perlfaq4.pod

```diff
@@ -1120,6 +1120,31 @@ signal that I missed something:
 
     print $string;
 
+=head2 Does Perl have anything like Ruby's #{} or Python's f string?
+
+Unlike the others, Perl allows you to embed a variable directly in a string C<"variable $variable">, when there isn't whitespace around the variable name you can add braces like C<"foo ${foo}bar">.
```

Comment:
Ruby allows this, too.
http://ruby-for-beginners.rubymonstas.org/bonus/string_interpolation.html